.eslintignore

```diff
@@ -6,5 +6,6 @@ node_modules/
 
 # TODO: don't ignore these examples once they're updated
 /examples/calendar-widget
+/examples/e-commerce
```

Comment:
We shouldn't ignore linting the examples, but only their built files: `.cache` and `dist`.

Reply:
@francoischalifour That's the comment on the line 7 right? Can we address that in other pull request?

Reply:
No, the end goal is to not ignore any examples. The ones listed here are the ones that have not been migrated to Parcel yet, and are not compliant with our lint configuration. `examples/e-commerce` is.

Ignoring `.cache` and `dist` (not `/dist`) should work.

Reply:
@francoischalifour Thanks for the feedback. Just updated the pull request.